<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>AddDefinition</class>
    <widget class="QWidget" name="AddDefinition">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>400</width>
                <height>300</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>AddDefinition</string>
        </property>
        <widget class="QPushButton" name="createButton">
            <property name="geometry">
                <rect>
                    <x>140</x>
                    <y>210</y>
                    <width>121</width>
                    <height>41</height>
                </rect>
            </property>
            <property name="text">
                <string>Create</string>
            </property>
        </widget>
        <widget class="QLineEdit" name="termLineEdit">
            <property name="geometry">
                <rect>
                    <x>110</x>
                    <y>110</y>
                    <width>201</width>
                    <height>21</height>
                </rect>
            </property>
        </widget>
        <widget class="QLineEdit" name="definitionLineEdit">
            <property name="geometry">
                <rect>
                    <x>110</x>
                    <y>150</y>
                    <width>201</width>
                    <height>21</height>
                </rect>
            </property>
        </widget>
        <widget class="QLabel" name="label">
            <property name="geometry">
                <rect>
                    <x>110</x>
                    <y>30</y>
                    <width>191</width>
                    <height>41</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <pointsize>14</pointsize>
                </font>
            </property>
            <property name="tabletTracking">
                <bool>false</bool>
            </property>
            <property name="text">
                <string>Add new definition</string>
            </property>
            <property name="alignment">
                <set>Qt::AlignCenter</set>
            </property>
        </widget>
        <widget class="QLabel" name="label_2">
            <property name="geometry">
                <rect>
                    <x>60</x>
                    <y>110</y>
                    <width>41</width>
                    <height>17</height>
                </rect>
            </property>
            <property name="text">
                <string>Term:</string>
            </property>
        </widget>
        <widget class="QLabel" name="label_3">
            <property name="geometry">
                <rect>
                    <x>40</x>
                    <y>150</y>
                    <width>61</width>
                    <height>17</height>
                </rect>
            </property>
            <property name="text">
                <string>Defition:</string>
            </property>
        </widget>
    </widget>
    <resources/>
    <connections/>
</ui>
